cmd/maya-exporter/app/collector/jivacollector.go

```diff
@@ -322,3 +329,26 @@ func (e *VolumeExporter) collect() error {
 	volumeUpTime.WithLabelValues(metrics1.Name, "iqn.2016-09.com.openebs.jiva:"+metrics1.Name, url).Set(metrics1.UpTime)
 	return nil
 }
+
+// Retry tries to initiates the connection with the socket. It retries
+// untill the connection is established.
+func (e *VolumeExporter) Retry() {
```

Comment:
RetryForever as the method name?

Reply:
Lets explain what happens if the socket can never be connected?
is it ok for the service to keep retrying. Lets explain in method comments. Perhaps a little bit on how the service
is expected to be managed.

Reply:
it will keep retry forever. Anyway comments can be added.